### PR TITLE
test(store): add more codegen tests [M-01]

### PR DESCRIPTION
**Pull request by holic**

### CODE REVIEW

File: packages/store/ts/storeEvents.test.ts
Comment (yonadaaa) [inline packages/store/ts/storeEvents.test.ts:1]: Should we add tests to cover more events than just `HelloStore`?

File: packages/store/test/constants.t.sol
Comment (yonadaaa) [inline packages/store/test/constants.t.sol:1]: Random thought: is it possible to implement this logic in TS (avoiding FFI) and read the constants from the Solidity files somehow?

**Review by yonadaaa — CHANGES_REQUESTED** [review]
Overall good, just I think we should be testing more events beyond `helloStoreEvent`, but maybe I missed that change

File: packages/store/ts/storeEvents.test.ts
Comment (yonadaaa) [reply packages/store/ts/storeEvents.test.ts:1]: Oh, I see now. Were we already testing this, so you just moved the test from `storeEventsAbi.test.ts`?

Comment (holic) [reply packages/store/ts/storeEvents.test.ts:1]: yep! I added the store events test to `storeEvents.test.ts` then found that it was essentially duplicating `storeEventsAbi.test.ts` so I removed it (which OZ also recommended)

File: packages/store/test/constants.t.sol
Comment (holic) [reply packages/store/test/constants.t.sol:1]: That was my initial thought/path but I didn't want to introduce the Solidity parser into this package. We were using the `vm.ffi` approach elsewhere already so I borrowed that:
https://github.com/latticexyz/mud/blob/main/packages/world/test/CoreModule.t.sol#L58-L71

**Pull request merged**